operations/finance.md

```diff
@@ -426,6 +426,8 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 | `GrossValue` | number | optional | Gross value including all taxes. |
 | `TaxValues` | array of [Tax Value](finance.md#tax-value) | optional | The tax values applied. |
 
+For most amounts, precision of values depends on `TaxPrecision` of [Enterprise](configuration.md#entreprise) or [Currency](configuration.md#currency) precision. But in some cases, precision can be higher.
```

Comment:
I would maybe just move it under the header dont we do it that way @jirrick ?

Reply:
Yep, that makes sense.